docs/fomega/package.yaml

```diff
@@ -62,4 +62,4 @@ benchmarks:
     - -fmax-simplifier-iterations=0
     - -threaded
     - -rtsopts
-    - '-with-rtsopts=-N'
\ No newline at end of file
+    - '-with-rtsopts=-N'
```

Comment:
An unrelated and accident end-of-file insertion. Do I need to bother to revert that?

Reply:
I don't care.